### DISCUSSION TITLE
Transaction History Not Syncing Across Zeus, RTL, and Alby on BTCPay Server

### DISCUSSION THREAD

**Discussion by jacobburrell** [btcpayserver/btcpayserver#5370] [Technical Support]
Hello everyone,

I'm experiencing an issue where transactions made through the Zeus mobile app are not showing up in the Ride The Lightning (RTL) web interface on my BTCPay Server. Interestingly, each interface (RTL, Zeus, and also Alby seems to maintain its own local view of transactions, which aren't in sync with each other.

I've already tried the following:

Checked for synchronization delays.
Confirmed the transaction in Zeus.
Reviewed server logs for errors.
Ensured that all software versions are up-to-date.
Is this a known issue? Does each interface indeed maintain its own local transaction history, and if so, how can I force them to sync with the actual state of the Lightning node?

Any guidance would be greatly appreciated. Thank you!

**Comment by dennisreimann** [comment]
All of the apps should more or less get the data from the underlying node, as far as I'm aware none of them keeps a cached or separate version of the data.

So the best way to debug this would be to use either the `./bitcoin-lncli.sh trackpayment PAYMENT_HASH_FROM_ZEUS` command to look up the particular transaction or `./bitcoin-lncli.sh listpayments` to see the list of recent transactions and see if it is also in there.

**Reply by jacobburrell** [reply]
Is there a GUI way of viewing all txs from the node in one place?

If they are all directly connecting to the same node, why don't they sync activity from others?

Trying `listpayments` works, all the txs from all sources show as expected.

Is the only way to resort to cl each time I want to see a complete tx list of the lighnting transactions happening on the LN node?

Is there a btcpayserver interface, plugin, or something else that does this via a GUI?

Restarting, waiting months, etc doesn't fix the issue, but yes `listpayments` directly does work.

**Reply by dennisreimann** [reply]
BTCPay doesn't have it's own views for most of Lightning (yet), but usually all those thirt-party tools work. Can you check whether or not maybe the app access keys expired and they potentially cannot refresh the information?

As said, I'm pretty sure that the ones you mention more or less directly use the same info you are getting via the CLI, as they use the LND API as well.

**Reply by jacobburrell** [reply]
In RTL's case I open it regularly from BTCPay's Lightning page under Wallets

<img width="389" alt="image" src="https://github.com/btcpayserver/btcpayserver/assets/61520543/14fdc029-abd3-4e00-a663-5446bc4e466b">

I can spend funds from any of the 3 connections mentioned... but I only see the tx in the place that made it. Yet the CLI does show all txs.

Any idea on next steps to debug?
